### ISSUE TITLE
ActiveMQ connection verifier doesn't honour provided values

### ISSUE THREAD

**Issue by dsimansk** [syndesisio/syndesis#622]
Regardless of provided values for Connection, the validation is always successful.

![screen shot 2017-12-06 at 09 48 05](https://user-images.githubusercontent.com/5637792/33652754-d4df5cf2-da6a-11e7-9ff7-642d4579395c.png)

Verifier log:
```
2017-12-06 08:49:24.410  WARN 1 --- [ XNIO-2 task-22] o.apache.activemq.broker.BrokerService   : Memory Usage for the Broker (1024mb) is more than the maximum available for the JVM: 228 mb - resetting to 70% of maximum available: 159 mb
2017-12-06 08:49:24.415  INFO 1 --- [ XNIO-2 task-22] o.apache.activemq.broker.BrokerService   : Using Persistence Adapter: MemoryPersistenceAdapter
2017-12-06 08:49:24.415  INFO 1 --- [ XNIO-2 task-22] o.apache.activemq.broker.BrokerService   : Apache ActiveMQ 5.15.2 (localhost, ID:syndesis-verifier-1-9hgqn-40167-1512545194998-0:498) is starting
2017-12-06 08:49:24.416  INFO 1 --- [  JMX connector] o.a.a.broker.jmx.ManagementContext       : JMX consoles can connect to service:jmx:rmi:///jndi/rmi://localhost:1099/jmxrmi
2017-12-06 08:49:24.416  INFO 1 --- [ XNIO-2 task-22] o.apache.activemq.broker.BrokerService   : Apache ActiveMQ 5.15.2 (localhost, ID:syndesis-verifier-1-9hgqn-40167-1512545194998-0:498) started
2017-12-06 08:49:24.416  WARN 1 --- [ XNIO-2 task-22] o.apache.activemq.broker.BrokerService   : Temporary Store limit is 51200 mb (current store usage is 0 mb). The data directory: /deployments only has 2590 mb of usable space. - resetting to maximum available disk space: 2590 mb
2017-12-06 08:49:24.417  INFO 1 --- [ XNIO-2 task-22] o.a.activemq.broker.TransportConnector   : Connector vm://localhost started
2017-12-06 08:49:24.426  INFO 1 --- [ XNIO-2 task-22] o.a.activemq.broker.TransportConnector   : Connector vm://localhost stopped
2017-12-06 08:49:24.426  INFO 1 --- [ XNIO-2 task-22] o.apache.activemq.broker.BrokerService   : Apache ActiveMQ 5.15.2 (localhost, ID:syndesis-verifier-1-9hgqn-40167-1512545194998-0:498) is shutting down
2017-12-06 08:49:24.427  INFO 1 --- [ XNIO-2 task-22] o.apache.activemq.broker.BrokerService   : Apache ActiveMQ 5.15.2 (localhost, ID:syndesis-verifier-1-9hgqn-40167-1512545194998-0:498) uptime 0.017 seconds
2017-12-06 08:49:24.427  INFO 1 --- [ XNIO-2 task-22] o.apache.activemq.broker.BrokerService   : Apache ActiveMQ 5.15.2 (localhost, ID:syndesis-verifier-1-9hgqn-40167-1512545194998-0:498) is shutdown
```

**Comment by dhirajsb** [comment]
The Broker log was misleading. It's coming from SB ActiveMQAutoConfiguration, which is starting an embedded Broker. I disabled it in application.yml and fixed the AMQ verifier in #651.

**Comment by dhirajsb** [comment]
Do watch out when using failover transport, since verification request will just timeout waiting for connection while failover transport keeps retrying by default.
